frontend/pages/hosts/ManageHostsPage/components/HostsFilterBlock/HostsFilterBlock.tsx

```diff
@@ -349,6 +351,8 @@ const HostsFilterBlock = ({
         label="Low disk space"
         tooltipDescription={TooltipDescription}
         onClear={() => handleClearFilter(["low_disk_space"])}
+        isSandboxMode={isSandboxMode}
+        sandboxPremiumOnlyIcon
```

Comment:
Can we add a longer close delay for this? I had a hard time getting my mouseover onto the tooltip to click learn more

<img width="581" alt="Screenshot 2023-04-24 at 2 49 43 PM" src="https://user-images.githubusercontent.com/71795832/234088727-bf32c919-4449-4345-a27b-4340af370845.png">

Reply:
Yes, coming right up...